### PR TITLE
Updated contribute/desktop _index.md

### PR DESCRIPTION
**Pull request by Rutam21**
#### Summary

The deprecated links were updated and the raw links were formatted for a better look and feel.

#### Ticket Link

NA

### CODE REVIEW

**Comment by Rutam21** [comment]
@cwarnermm Please review and add the necessary labels for Docathon.

File: site/content/contribute/desktop/_index.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ date: 2018-01-02T10:36:34-05:00
 weight: 7
 ---
 
-The Mattermost desktop app is an [Electron](https://electronjs.org/) wrapper around the [web app](/contribute/webapp) project. It lives in the [mattermost/desktop](https://github.com/mattermost/desktop) repository. The desktop app runs on Windows, Linux, and macOS.
+The Mattermost desktop app is an [Electron](https://electronjs.org/) wrapper around the [web app](https://github.com/mattermost/mattermost-developer-documentation/tree/master/site/content/contribute/webapp) project. It lives in the [mattermost/desktop](https://github.com/mattermost/desktop) repository. The desktop app runs on Windows, Linux, and macOS.
```
Comment (justinegeffen) [inline site/content/contribute/desktop/_index.md:9]:
```suggestion
The Mattermost desktop app is an [Electron](https://electronjs.org/) wrapper around the [web app](/contribute/webapp) project. It lives in the [mattermost/desktop](https://github.com/mattermost/desktop) repository. The desktop app runs on Windows, Linux, and macOS.
```

**Review by justinegeffen — APPROVED** [review]
Thank you for your contribution! :)

**Review by cwarnermm — APPROVED** [review]
Thank you for your docs contribution!

**Pull request merged**